debian/changelog

```diff
@@ -1,3 +1,9 @@
+daos (2.5.100-16) unstable; urgency=medium
+  [ Li Wei ]
+  * Update raft to 0.11.0-1416.g12dbc15
```

Comment:
should this instead refer to the version as: 0.11.0-1.415.g12dbc15 ? I ask only because I'm comparing to a previous change by @brianjmurrell in this file when updating raft to 0.10.1-2.411.gefa15f4 and supposing that this commit might want to use a consistent version string here in the description.

And also curious if the daos.spec file changelong version numbers should be consistent with what is reported in the debian/changelog.

Reply:
This debian/changelog file is for debian/control; it is not for the RPM side of things. So I think here we should refer to the version string in debian/control.

Reply:
I'm not really sure why there is no dot after the `1` in `1416`.  It could be some artifact of debian's versioning that I am not recalling off-hand.  Or it could be a bug.  Not terribly urgent in either case.  Indeed, we'll be in a good place if this has risen to the top of the priority list.  :-)